whirlpool/washerdryer.py

```diff
@@ -79,6 +81,59 @@ class MachineState(Enum):
     MachineState.SystemInit: ATTRVAL_MACHINE_STATE_SYSTEMINIT,
 }
 
+MACHINE_STATE_MAP_PRETTY = {
+    MachineState.Standby: "Standby",
+    MachineState.Setting: "Setting",
+    MachineState.DelayCountdownMode: "Delay - Countdown",
+    MachineState.DelayPause: "Delay - Paused",
+    MachineState.SmartDelay: "Delay - Completion time specified",
+    MachineState.SmartGridPause: "Delay - Smart grid pause",
+    MachineState.Pause: "Paused",
+    MachineState.RunningMainCycle: "Running - Main cycle",
+    MachineState.RunningPostCycle: "Running - Post cycle",
+    MachineState.Exceptions: "Error - Exception",
+    MachineState.Complete: "Complete",
+    MachineState.PowerFailure: "Error - Power failure",
+    MachineState.ServiceDiagnostic: "Diagnostic - Service",
+    MachineState.FactoryDiagnostic: "Diagnostic - Factory",
+    MachineState.LifeTest: "Diagnostic - Life test",
+    MachineState.CustomerFocusMode: "Special mode - Customer Focus",
+    MachineState.DemoMode: "Special Mode - Demo",
+    MachineState.HardStopOrError: "Error - Hard stop",
+    MachineState.SystemInit: "Initializing",
+}
+
+class WashCycleState (Enum):
+    Sensing = False
+    Filling = False
+    Soaking = False
+    Washing = False
+    Rinsing = False
+    Spinning= False
```

Comment:
we should not have an enum class with all the values being equal (False). I'm not sure about the approach done here regarding those values.